src/app/core/services/cluster.service.spec.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ClusterService} from './cluster';
+import {HttpClientTestingModule, HttpTestingController} from '@angular/common/http/testing';
+import {TestBed} from '@angular/core/testing';
+import {MatDialogModule} from '@angular/material/dialog';
+import {AppConfigMockService} from '@test/services/app-config-mock';
+import {AppConfigService} from '@app/config.service';
+import {lastValueFrom} from 'rxjs';
+
+fdescribe('ClusterService', () => {
+  let service: ClusterService;
+  let httpController: HttpTestingController;
+
+  const mockProjectID = 'fake-project-id';
+  const mockClusterID = 'fake-cluster-id';
+  const mockMachineDeploymentID = 'fake-machine-deployment-id';
+  const mockRestRoot = 'mock-rest-root';
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ClusterService, {provide: AppConfigService, useClass: AppConfigMockService}],
+      imports: [HttpClientTestingModule, MatDialogModule],
+    });
+    httpController = TestBed.inject(HttpTestingController);
+    service = TestBed.inject(ClusterService);
```

Comment:
is it equivalent to? `fixture.debugElement.injector.get(ClusterService) as any;` to inject service?

Reply:
It looks equivalent, but tbh I have always used  `TestBed.inject()` instead of `ixture.debugElement.injector.get(ClusterService) as any;`